Editor/Window/AwsUserProfilesPage.cs

```diff
@@ -116,7 +116,15 @@ private void LocalizeText()
 
         private void SetupButtonCallbacks()
         {
-            _container.Q<Button>("UserProfilePageAccountCardNoAccountButton").RegisterCallback<ClickEvent>(_ => OpenLink("")); // TODO: Investigate correct URL
+            _container.Q<Button>("UserProfilePageAccountCardNoAccountButton")
+                .RegisterCallback<ClickEvent>(_ => OpenLink(Urls.CreateAwsAccountLearnMore));
+            _container.Q<VisualElement>("UserProfilePageBootstrapHelpLink")
+                .RegisterCallback<ClickEvent>(_ => OpenLink(Urls.S3BootstrapHelp));
+            _container.Q<VisualElement>("UserProfilePageCompletedBootstrapHelpLink")
+                .RegisterCallback<ClickEvent>(_ => OpenLink(Urls.S3BootstrapHelp));
+            _container.Q<VisualElement>("UserProfilePageAccountNewProfileHelpLink")
+                .RegisterCallback<ClickEvent>(_ => OpenLink(Urls.AwsIamDocumentation));
+
```

Comment:
I don't see the need for a utility in this case. Can we use Application.OpenURL inline where required? Or alternatively we should use common utilities that all files can we use so we don't create a utility per file.

Reply:
Removed utility